### ISSUE TITLE
Package download failed

### ISSUE THREAD

**Issue by Agent57** [Borcioo/laravel-table-builder#1]
Hi,

I was interested to review your table builder package after seeing your post on the InertisJS Discord server. Unfortunately, I'm failing before I even get started.

I have a clean install of Laravel 11 and added your github repo to my composer.json. When I run the composer install command I get the following output...
```
$ composer require borek/laravel-table-builder:dev-main

In CurlDownloader.php line 641:
                                                                                                                          
  The "https://api.github.com/repos/borek/laravel-table-builder" file could not be downloaded (HTTP/2 404 ):              
  {"message":"Not Found","documentation_url":"https://docs.github.com/rest/repos/repos#get-a-repository","status":"404"}  
                                                                                                                          

require [--dev] [--dry-run] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--fixed] [--no-suggest] [--no-progress] [--no-update] [--no-install] [--no-audi
t] [--audit-format AUDIT-FORMAT] [--update-no-dev] [-w|--update-with-dependencies] [-W|--update-with-all-dependencies] [--with-dependencies] [--with-all-dependencies] [--ignore
-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [-m|--minimal-changes] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--] [<packages>...]
```

Not sure if I'm missing something, but if you can provide any assistance I'd like to try it out.

In my current project I couldn't find a 3rd party table component that provided the features and flexibility that I desired, so ended up creating my own.

![image](https://github.com/user-attachments/assets/e5fce0c6-7e2c-4d2d-afd7-cc046f445d6f)

Of course, my implementation is very different to yours, but if my experience can help in any way then I'm happy to assist.

Thanks,
Lee

**Comment by Borcioo** [comment]
Fixed in commit [c9ade5c6924e44244feea065d235776c33502674].

The issue has been resolved by:
1. Wrond repo url in readme.md

The changes are available in the latest version. Please update and let me know if you encounter any issues.

**Comment by Agent57** [comment]
It's a slightly different error message, but I'm not getting any further...
```
$ composer require borek/laravel-table-builder:dev-main
./composer.json has been updated
Running composer update borek/laravel-table-builder
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires borek/laravel-table-builder, it could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```
